ftd/t/js/80-or-type-constant.ftd

```diff
@@ -0,0 +1,15 @@
+-- or-type button-type:
+
+-- constant integer small: 1
+
+-- constant integer medium: 2
+
+-- constant integer large: 3
+
+-- end: button-type
+
+
+-- button-type b: small
+
+-- ftd.text: !Smol
+text if { b == button-type.small }: Smol
```

Comment:
Why not this?

```suggestion
text if { b == small }: Smol
```

In line 12 we used just `small`, so we should use it in 15 also.

Reply:
Okay, got it.